app/views/admin/loans/questionnaires/_breakeven_table_question.html.slim

```diff
@@ -25,6 +25,24 @@
         = render 'admin/loans/questionnaires/breakeven_product_row', question: question
 
   .table-container
+    h4.title = t("loan.breakeven.rampup")
+
+    table.table.table-condensed.table-striped.table-bordered.editable-table.breakeven-table-rampup data-table="rampup"
+      thead
+        tr
+          th = t("loan.breakeven.periods")
+          th = t("loan.breakeven.units")
+      tbody
+        / Hidden row is used to generate new rows
+        / Is at top instead of bottom to not affect striped rows
+        tr.ui-state-default data-group="rampup" class="hidden"
+
+        - if data
+          = render 'admin/loans/questionnaires/breakeven_rampup_row', f: f, periods: data[:periods], units: data[:units], question: question
+        - else
+          = render 'admin/loans/questionnaires/breakeven_rampup_row', f: f, question: question
```

Comment:
I was debating with myself if I should do that or not. I like to be more explicit with what gets passed in, but you are right we would save some complexity.